Rubberduck.VBEEditor/SafeComWrappers/VB/Enums/VBEKind.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rubberduck.VBEditor.SafeComWrappers
+{
+    public enum VBEKind
+    {
+        /// <summary>Embedded VB editor (Visual Basic for Applications).</summary>
+        Embedded,
```

Comment:
nitpick: I'd have it like this:

```csharp
public enum VBEKind
{
    /// <summary>
    /// Embedded VB editor (Visual Basic for Applications).
    /// </summary>
    Embedded,

    /// <summary>
    /// Standalone VB editor (Visual Basic).
    /// </summary>
    Standalone,
}
```

Reply:
Yep agreed, that's a personal style choice of mine that I accept is non-standard. I just missed this (too much context switching between home and work projects atm).

Will change.

Reply:
To further nitpick: can we go for `Hosted` instead of `Embedded` then? :wink:

Reply:
Sure :-)